api/rbac/v1alpha1/role_types.go

```diff
@@ -32,8 +30,11 @@ type ResourceDetails struct {
 	Verbs        []string `json:"verbs,omitempty" protobuf:"bytes,3,rep,name=verbs"`
 }
 
+type Claim struct {
+	Name   string
+	Values []string
+}
+
 type UserClaims struct {
```

Comment:
Since this type encapsulates only an array of Claims, I am wondering if we really need both UserClaims _and_ Claims or whether we can drop one of them.

Reply:
Yes makes sense, I have removed the Claim struct and just got the Role to use an array of UserClaims